wled00/set.cpp

```diff
@@ -247,7 +247,10 @@ void handleSettingsSet(AsyncWebServerRequest *request, byte subPage)
     notifyAlexa = request->hasArg(F("SA"));
     notifyHue = request->hasArg(F("SH"));
     notifyMacro = request->hasArg(F("SM"));
-    notifyTwice = request->hasArg(F("S2"));
+
+    t = request->arg(F("UR")).toInt();
+    if (t>0) ubUdpNumRetries = t;
```

Comment:
I would limit retries to 10 or some other low number otherwise a user may enter 999999 retries or more.